### PR TITLE
make to-admin-csv script

### PR DESCRIPTION
**Pull request by maxgrossman**
- make to-admin-csv, a refactor to-admin-geojson that writes admin level dumps to csv.

### CODE REVIEW

File: scripts/to-admin-geojson.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+// for each feature, push obj to mappedFeatures w/adminCode and feature
+fs.createReadStream(process.argv[2])
+  .pipe(parser)
+  .on('data', (feature) => {
+    const rollupObj = {};
+    // only push features to mappedFeatures where or_vpromms exist.
+    if (feature.properties.or_vpromms) {
+      const adminID = feature.properties.or_vpromms.slice(0, 2);
+      rollupObj['admin'] = adminID;
+>>>>>>> d3e032927b34708b09bf4c985013d5ad4656d893
```
Comment (mileswwatkins) [inline scripts/to-admin-geojson.js]: You've got a diff markup committed.

**Review by mileswwatkins — CHANGES_REQUESTED** [review]
`develop` is significantly behind `master`, and it looks like there are some `git diff` artifacts in your code that would break it; can you fix these, confirm it's running on your end, and then ping me to look over the code?

**Comment by maxgrossman** [comment]
@miles - updated develop to master, then this branch to develop. Also reran and output is as expected.

File: scripts/make-analysis-and-display.sh [hunk 2]
```diff
@@ -58,13 +62,16 @@ mapbox upload \
     "${MAPBOX_ACCOUNT}.vietnam-conflated" \
     ".tmp/conflated.mbtiles"
 
+echo "There is currently no analytics processing, but it would be insterted here"
+
 echo "Dump un-conflated, by-province data to S3, for public consumption"
-mkdir .tmp/by-province-name
-./to-admin-geojson.js .tmp/network-merged.geojson .tmp/by-province-name
+mkdir .tmp/by-province-id
+# ./to-admin-geojson.js .tmp/network-merged.geojson .tmp/by-province-id
```
Comment (mileswwatkins) [inline scripts/make-analysis-and-display.sh]: We still want to run and dump the GeoJSON, just _additionally_ dump the CSVs with WKT. (You'll need to `chmod` the file permissions of `scripts/to-admin-geojson.js` back to `755`.)

File: scripts/to-admin-csv.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
+'use strict';
+// writes road geojsons grouped by admin.
```
Comment (mileswwatkins) [inline scripts/to-admin-csv.js]: CSVs

File: scripts/make-analysis-and-display.sh (hunk 2 above)
Comment (mileswwatkins) [reply scripts/make-analysis-and-display.sh]: (I also think you'll need to add subadmin codes to that script as well, then.)

**Review by mileswwatkins — CHANGES_REQUESTED** [review]
Small tweaks, then you can self-merge once you clear them.

I think that the on-end code block is overengineered; I think using some native methods and a more general CSV serialization library, you could accomplish the same result more directly. But no time to tweak it now :)

**Pull request merged**